pkgs/applications/science/biology/multiqc/default.nix

```diff
@@ -0,0 +1,37 @@
+{ python3Packages, fetchPypi, lib }:
+
+python3Packages.buildPythonApplication rec {
+  version = "1.15";
+  pname = "multiqc";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-zlNZoSImz0zjcsb9rRQs/irnUB/6l6x6q1RM7U216jw=";
+  };
+  doCheck = false;
```

Comment:
Any chance to run the tests (at least partially), or to find out why it won't work?